src/gui/dialogs/addon/manager.cpp

```diff
@@ -391,6 +396,50 @@ void addon_manager::pre_show(window& window)
 	connect_signal_notify_modified(type_filter,
 		std::bind(&addon_manager::apply_filters, this));
 
+	// Language filter
+	// Clear current language filter vector, lest all entries are shown twice
+	addon_manager::language_filter_types_.clear();
+	addon_manager::language_filter_types_.push_back(std::pair<int, std::string>(0, "en_GB"));
+
+	// Prepare shown languages, source all available languages from the addons themselves
+	std::vector<std::string> languages_available;
+	for(const auto& a : addons_) {
+		for (const auto& b : a.second.locales) {
+			if (!(b.rfind("en_GB", 0) == 0)) {
+				languages_available.push_back(b);
+			}
+		}
+	}
+	std::sort(languages_available.begin(), languages_available.end());
+	auto last = std::unique(languages_available.begin(),languages_available.end());
+	languages_available.erase(last, languages_available.end());
+	// Erase pt value, since otherwise Portugues do Brasil will be shown twice in the language list
+	(void)std::remove(languages_available.begin(), languages_available.end(), "pt");
+
+	language_filter_types_[0].second = langcode_to_string(language_filter_types_[0].second) != "" ? langcode_to_string(language_filter_types_[0].second) : "English (GB)";
```

Comment:
Again, `en_GB` should not be hardcoded. Remove this bit.

Reply:
:ok_hand: